e2e/Dockerfile

```diff
@@ -6,13 +6,13 @@ WORKDIR /stuff
 COPY package.json .
 COPY package-lock.json .
 
-RUN git clone https://github.com/poanetwork/poa-bridge-contracts.git
+RUN git clone https://github.com/ColuLocalNetwork/poa-bridge-contracts.git
```

Comment:
After https://github.com/poanetwork/poa-bridge-contracts/pull/140 is merged we should update repository reference